Gemfile.rails32

```diff
@@ -1,10 +1,12 @@
-group :test do
-  gem 'json-jruby', :platforms => :jruby
```

Comment:
I removed this because it's part of Ruby's standard library, and the `json` gem is supported directly in JRuby these days.